Documentation/ceph-dashboard.md

```diff
@@ -59,6 +59,26 @@ To get the dashboard to use hyperlinks that include your prefix, you can set the
       urlPrefix: /ceph-dashboard
 ```
 
+The port that the dashboard is served on may be changed from the defaults using
+the `port` settings. The corresponding K8s service exposing the port will
+automatically be updated:
+```yaml
+  spec:
+    dashboard:
+      port: 43219
+```
+
+The dashboard may also be served without SSL (useful for when you deploy the
+dashboard behind a proxy already served using SSL) by setting the `ssl` option
+to be false. Note that SSL is not currently supported in Luminous, and can be
```

Comment:
How about this wording for the last sentence?
"Note that the ssl setting will be ignored in Luminous as well as Mimic 13.2.2 or older where it is not supported"